controllers/defaults/placements.go

```diff
@@ -176,6 +182,26 @@ func getWeightedPodAffinityTerm(weight int32, selectorValue ...string) corev1.We
 	return WeightedPodAffinityTerm
 }
 
+func getMdsWeightedPodAffinityTerm(weight int32, selectorValue ...string) corev1.WeightedPodAffinityTerm {
+	WeightedPodAffinityTerm := corev1.WeightedPodAffinityTerm{
```

Comment:
can return this directly, no need for a var
```suggestion
	return corev1.WeightedPodAffinityTerm{
```